sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/Customizations/FactoryVSTSConfiguration.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Microsoft.Azure.Management.DataFactory.Models
+{
+    public partial class FactoryVSTSConfiguration : FactoryRepoConfiguration
+    {
+        /// <summary>
+        /// Initializes a new instance of the FactoryVSTSConfiguration class.
+        /// </summary>
+        /// <param name="accountName">Account name.</param>
+        /// <param name="repositoryName">Repository name.</param>
+        /// <param name="collaborationBranch">Collaboration branch.</param>
+        /// <param name="rootFolder">Root folder.</param>
+        /// <param name="projectName">VSTS project name.</param>
+        /// <param name="lastCommitId">Last commit id.</param>
+        /// <param name="tenantId">VSTS tenant id.</param>
+        public FactoryVSTSConfiguration(string accountName, string repositoryName, string collaborationBranch, string rootFolder, string projectName, string lastCommitId = default(string), string tenantId = default(string))
```

Comment:
I don't think it's a right way to minimize the breaking change. If the customer calls this method in the previous version like:
```csharp
var test = FactoryVSTSConfiguration("accountName", "repositoryName", "collaborationBranch", "rootFolder", "projectName")
```
There is no doubt that this call will become an ambiguous call in the new version. Therefore I think a more appropriate customization should be:
```csharp
public FactoryVSTSConfiguration(string accountName, string repositoryName, string collaborationBranch, string rootFolder, string projectName, string lastCommitId, string tenantId)
```

Reply:
fix